prog/plugin/main.go

```diff
@@ -26,6 +26,7 @@ var version = "(unreleased version)"
 func main() {
 	var (
 		justVersion      bool
+		cniNet, cniIpam  bool
```

Comment:
I've never seen us do this anywhere else - why here? Why not `address, meshAddress string`?